.github/workflows/release-actions.yml

```diff
@@ -126,16 +126,36 @@ jobs:
           python-version: |
             3.10
             3.6
+      - name: Detect project manager
+        id: manager
+        run: |
+          if [ -f "poetry.lock" ]; then
+            echo "name=poetry" >> $GITHUB_OUTPUT
+          elif [ -f "uv.lock" ]; then
+            echo "name=uv" >> $GITHUB_OUTPUT
+          else
+            exit 1
+          fi
```

Comment:
Hier wird zwar eine Fallunterscheidung für uv statt tox vorbereitet aber in Zeilem 166 und 168 wird mit **immer** mit `tox` gearbeitet, oder?

Das ist aber jetzt ja gar nicht mehr installiert, oder?